ComposureWatch/src/Components/Body.jsx

```diff
@@ -1,7 +1,5 @@
 const Body = (props) => {
-  const classes =
-    "w-screen h-screen text-[#1d253a]p-0 m-0 box-border block" +
-    props.className;
+  const classes = "text-[#1d253a] p-0 m-0 box-border block" + props.className;
```

Comment:
You might want to add a space after "block" here to ensure the passed-in prop gets added as its own class and not appended to "block", so they don't have to pass a space in with their class name.

For example, currently if I passed in `<Body className="boom" />` it would give me `blockboom`